src/store/entities/Member.java

```diff
@@ -2,4 +2,122 @@
 
 public class Member {
 
+	private String name;
+	private String address;
+	private String phoneNumber;
+	private String dateJoined;
+	private double feePaid;
+	private int id;
+
+	private static int idCounter = 1;
+
+	Member(String name, String address, String phoneNumber, String dateJoined, double feePaid) {
+		this.name = name;
+		this.address = address;
+		this.phoneNumber = phoneNumber;
+		this.dateJoined = dateJoined;
+		this.feePaid = feePaid;
+		this.id = idCounter++;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getAddress() {
+		return address;
+	}
+
+	public void setAddress(String address) {
+		this.address = address;
+	}
+
+	public String getPhoneNumber() {
+		return phoneNumber;
+	}
+
+	public void setPhoneNumber(String phoneNumber) {
+		this.phoneNumber = phoneNumber;
+	}
+
+	public String getDateJoined() {
+		return dateJoined;
+	}
+
+	public void setDateJoined(String dateJoined) {
+		this.dateJoined = dateJoined;
+	}
+
+	public double getFeePaid() {
+		return feePaid;
+	}
+
+	public void setFeePaid(double feePaid) {
+		this.feePaid = feePaid;
+	}
+
+	public int getId() {
+		return id;
+	}
+
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + ((address == null) ? 0 : address.hashCode());
+		result = prime * result + ((dateJoined == null) ? 0 : dateJoined.hashCode());
+		long temp;
+		temp = Double.doubleToLongBits(feePaid);
+		result = prime * result + (int) (temp ^ (temp >>> 32));
+		result = prime * result + id;
+		result = prime * result + ((name == null) ? 0 : name.hashCode());
+		result = prime * result + ((phoneNumber == null) ? 0 : phoneNumber.hashCode());
+		return result;
+	}
+
+	@Override
+	public boolean equals(Object obj) {
```

Comment:
Looks good, man. But you know that the professor doesn't like field names to be "obj".